oidc.go

```diff
@@ -45,6 +45,11 @@ func (h *Headscale) initOIDC() error {
 			return err
 		}
 
+		scope := h.cfg.OIDC.Scope
+		if len(scope) == 0 {
+			scope = []string{oidc.ScopeOpenID, "profile", "email"}
+		}
```

Comment:
Maybe it would be nicer to set this as a default instead? That way we have all "default" logic in the config part and not special logic polluting the code.